app/controllers/houston/dashboards/staging_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Houston::Dashboards::StagingController < ApplicationController
   layout "houston/dashboards/dashboard"
-  STAGING_PROJECTS = %w(members unite ledger bsb)
+  STAGING_PROJECTS = %w(members unite ledger bsb members-dav)
```

Comment:
This is shaping up nicely!!

I just published houston-core 0.5.5 which:

  1. Adds `#body` to `Github::PullRequest` and syncs that
  2. Adds the `.labeled` scope I was talking about (e.g. `Github::PullRequest.labeled "on-staging"`)
  3. Adds `#props` to  `Github::PullRequest`, a jsonb column which Houston Core provides so that modules can add arbitrary attributes (like `sequence` when we get around to drag-sorting the upcoming pull request)